README.md
# dotFilesFourtek

`dotFiles` to be used at repo. 

These are in a state of flux! 

Here, be dragons :)


eklavya-mac/history.txt
   15  mkdir code-snippets
   16  mv natural-lib-snippet.clj ./code-snippets
   17  gem install pry
   18  which pry
   19  git commit -m "before adding python and vim wishlists"
   20  git commit -m "added the new wishlists"
   21  brew install git-lfs
   22  rm -rf /Volumes/HARSHIT/colloquial_kazakh.pdf 
   23  rm -rf /Volumes/HARSHIT/uni_page_TXT 
   24  rm -rf /Volumes/HARSHIT/Dominating\ TED\ Talks 
   25  rm -rf /Users/eklavya/.Trash/apps.txt /Users/eklavya/.Trash/boilerplates /Users/eklavya/.Trash/Bootstrap\ HTML5\ Responsive\ Templates /Users/eklavya/.Trash/clear_text /Users/eklavya/.Trash/colloquial_kazakh_jpg /Users/eklavya/.Trash/contributors.txt /Users/eklavya/.Trash/decoding_jp_en_HTML\ copy.py /Users/eklavya/.Trash/Documentation_DASH /Users/eklavya/.Trash/dotFiles /Users/eklavya/.Trash/electron /Users/eklavya/.Trash/electron-role-models /Users/eklavya/.Trash/MemGuild /Users/eklavya/.Trash/My\ Apps\ _\ Wunderlist\ Developer_files /Users/eklavya/.Trash/My\ Apps\ _\ Wunderlist\ Developer.html /Users/eklavya/.Trash/NODE /Users/eklavya/.Trash/projects_active /Users/eklavya/.Trash/projects_ideas /Users/eklavya/.Trash/PYTHON /Users/eklavya/.Trash/python\ 11.45.32\ pm /Users/eklavya/.Trash/rclone-v1.30-osx-amd64 /Users/eklavya/.Trash/ruby /Users/eklavya/.Trash/sample\ notepad\ apps /Users/eklavya/.Trash/shell /Users/eklavya/.Trash/sublime-alda-master /Users/eklavya/.Trash/Tutorials\ for\ electron /Users/eklavya/.Trash/Web\ Templates /Users/eklavya/.Trash/WebDev /Users/eklavya/.Trash/WebDev\ 11.18.36\ pm 
   26  cat repl-port
   27  mv profiles.clj profiles.clj.original
   28  cd repository/cider/cider-nrepl
   29  cd 0.14.0-SNAPSHOT
   30  vim profiles.clj.original
   31  mv profiles.clj.original profiles.clj
   32  repl_up
   33  npm install tern -g
   34  history | grep 'npm'
   35  npm link tern
   36  brew install pandoc -j4
   37  brew cask install mactex
   38  brew install sbcl
   39  drracket
   40  /Applications/Racket\ v6.5/bin/scribble 
   41  source .zshrc
   42  _ rm -rf /Volumes/HARSHIT/proton-master.zip /Volumes/HARSHIT/ReactMVA-master.zip 
   43  cd Projects/bi
   44  vim example.js
   45  node example.js
   46  git commit -m "will be working in branches of this to port the electron-sample-apps package"
   47  git commit -m "moved helloworld to this folder and branch"
   48  Electron helloworld
   49  git commit -m "mission accomplished"
   50  git checkout master 
   51  Electron ../pwd
   52  ../pwd
   53  ..pwd
   54  cd "/Users/eklavya/Projects/bitworld/electron-sample-apps-master/menus" && clear
   55  Electron menus
   56  git branch sample-menus
   57  git checkout sample-menus 
   58  git commit -m "working on sample-menus"
   59  history | grep 'Electron'
   60  Electron cljs-electron-master/menus
   61  git commit -m "sanitized the ui.core.cljs"
   62  git branch sample-helloworld 1-samplehelloworld
   63  git branch 1-sample-helloworld samplehelloworld
   64  which node
   65  cd "/Users/eklavya/Projects/TheTalkingApes/TTA_Projects/SAILS_learning_with_text/lwt_v_1_6_1/files-INC" && clear
   66  vim utilities.inc.php
   67  open demo.gif
   68  cd "/Users/eklavya/Projects/bitworld/electron-sample-apps-master/helloworld-sharedobj" && clear
   69  Electron ../helloworld-sharedobj
   70  cd "/Users/eklavya/Projects/bitworld/CLOJURE/browser-repl" && clear
   71  rlwrap java -cp cljs.jar:src clojure.main browser-repl.clj
   72  cd twin-repls
   73  cd ../browser-repl
   74  java -cp cljs.jar:src clojure.main browser-repl.clj
   75  mv browser-repl.clj browser_repl.clj
   76  java -cp cljs.jar:src clojure.main node_repl.clj
   77  cd "/Users/eklavya/Projects/bitworld/CLOJURE/browser_repl" && clear
   78  lein cljsbuild once browser_repl
   79  cd browser_repl
   80  echo " <html>\n    <body>\n        <script type="text/javascript" src="out/main.js"></script>\n    </body>\n</html>">>index.html
   81  cd "/Users/eklavya/Projects/bitworld/CLOJURE/browser_repl/out" && clear
   82  cd /Users/eklavya/Projects/bitworld/CLOJURE/hello-world/browser_repl_2/hello_world; clear; pwd
   83  mkdir -p src/hello_world;touch src/hello_world/core.cljs
   84  vim build.clj
   85  cd /Users/eklavya/Projects/bitworld/CLOJURE/browser_repl; clear; pwd
   86  history | grep 'repl'
   87  vim repl.clj
   88  java -cp cljs.jar:src clojure.main browser_repl.clj
   89  rlwrap java -cp cljs.jar:src clojure.main repl.clj
   90  cd /Users/eklavya/Projects/bitworld/CLOJURE/browser_repl_2; clear; pwd
   91  nf
   92  jjs
   93  nashorn
   94  /usr/local/jjs
   95  /usr/local/bin/jjs
   96  /usr/bin/jjs
   97  history | grep 'jjs'
   98  jrunscript
   99  cljs
  100  /bin/cljsc
  101  usr/bin/cljsc
  102  cd usr
  103  find cljsc
  104  npm | grep 'express'
  105  npm | grep "express"
  106  cd /bin
  107  cd src/hello_world/
  108  vim ./.lein
  109  cd reagent
  110  cd 0.6.0-alpha
  111  emacs --insecure
  112  vim .spa
  113  java -cp cljs.jar:src clojure.main build.clj
  114  cd "/Users/eklavya/Projects/bitworld/CLOJURE/SNIPPETS" && clear
  115  cd "/Users/eklavya/Projects/bitworld/CLOJURE/browser_repl/code-snippets" && clear
  116  cd Projects/bitworld/clojURE
  117  lein new figwheel fig_browser
  118  cd "/Users/eklavya/Projects/bitworld/cljs-electron-master" && clear
  119  git commit -m "work in progress"
  120  cd Projects/bitworld/electron-sample-apps-master
  121  Electron tray
  122  git branch sample-tray
  123  git commit -m "started a port of tray app"
  124  g s
  125  git commit -m "we have a proper UI file here, but in actual app there's only a single one"
  126  cd "/Users/eklavya/Projects/bitworld/cljs-electron-master/electron_src/electron" && clear
  127  cd Projects/bitworld/CLOJURE/SNIPPETS

eklavya-mac/profiles.clj
{ :user  {
          :dependencies [
[kioo "0.4.2"] 
[hipo "0.5.2"]


                         [org.clojure/clojure "1.9.0-alpha17"]
                         [org.clojure/clojurescript "1.9.854"]
                         [rewrite-cljs "0.4.3"]
                         [hiccups "0.3.0"]
                         [com.cemerick/piggieback "0.2.1"]
                         [semantic-csv "0.1.0"]
                         [org.clojure/core.async "0.2.385"]
                         [com.jsyn/jsyn "16.7.8"]
                         [aysylu/loom "0.6.0"]
                         [com.cognitect/transit-cljs "0.8.239"]
                         [org.clojure/java.classpath "0.2.3"]
                         [com.cognitect/transit-clj "0.8.293"]
                         [com.itextpdf/itextpdf "5.5.10"]
                         [edu.stanford.nlp/stanford-corenlp "3.6.0"]
                         [reagent "0.6.0-rc"]
                         [org.apache.commons/commons-compress "1.5"]
                         [overtone "0.10.1"]                 ;                         [org.apache.commons/commons-vfs "2 2.1"]
                         [com.google.code.gson/gson "2.8.0"]
                         [com.google.jimfs/jimfs "1.0"]
                         [org.imgscalr/imgscalr-lib "4.2"]
   
;                         [net.sourceforge.tess4j/tess4j "3.2.1"]
   
                         [net.bytebuddy/byte-buddy "1.5.3"]
                         [com.twelvemonkeys.imageio/imageio-core "3.3"]
                         [org.apache.shiro/shiro-core "1.3.2"]
                         [jline/jline "2.12"]
                         [org.apache.commons/commons-lang3 "3.5"]
                         [commons-io/commons-io "2.5"]
                         [clansi "1.0.0"]
                         [rewrite-clj "0.6.0"]
                         [lein-ancient "0.6.10"]
                         [org.clojure/core.typed "0.3.23"]
                         [org.clojure/core.logic "0.8.10"]
                         [com.google.jimfs/jimfs "1.1"]
                         [devcards "0.2.1-7"]
                         [binaryage/dirac "0.6.4"]
                         [binaryage/devtools "0.8.1"]
                         [cljsjs/d3 "4.2.0-0"]
                         [hickory "0.6.0"]
                         [hiccup "1.0.5"]
                         [org.clojure/algo.generic "0.1.2"]
                         [org.clojure/core.match "0.3.0-alpha4"]
                         [org.clojure/core.cache "0.6.5"]
                         [org.clojure/core.memoize "0.5.8"]
                         [instaparse "1.4.3"]
                         [com.google.guava/guava "19.0"]
                         [com.ibm.icu/icu4j "57.1"]
                         [korma "0.4.3"]
                         [funcool/cats "2.0.0"]
                         [quil "2.4.0"]
                         [automat "0.2.0"]
                         [frak "0.1.6"]
                         [clojureverbalexpressions "0.2.1"]
                         [funcool/cuerdas "2.0.0"]
                         [cljsjs/semantic-ui "2.2.2-0"]
                         [io.aviso/pretty "0.1.30"]
                         [spyscope "0.1.5"]
                         [net.cgrand/seqexp "0.6.0"]
                         [cljsjs/chance "0.7.3-0"]
                         [funcool/promesa "1.8.1"]
                         [org.clojure/math.combinatorics "0.1.3"]
                         [org.clojure/test.check "0.9.0"]
                         [org.clojure/tools.analyzer "0.6.9"]
                         [org.clojure/tools.cli "0.3.5"]
                         [org.clojure/tools.macro "0.1.2"]
                         [org.clojure/tools.namespace "0.2.11"]
                         [org.clojure/data.generators "0.1.2"]
                         [ru.lanwen.verbalregex/java-verbal-expressions "1.4"]
                         [org.clojure/data.csv "0.1.3"]
                         [org.clojure/data.json"0.2.6"]
                         [org.apache.commons/commons-math3 "3.6.1"]
                         [ re-frame "0.8.0"]]


          :plugins [
;                    [nightlight/lein-nightlight "1.6.1"]                    ;                    [venantius/ultra "0.5.0"]
 ;                   [lein-marginalia "0.9.0"]
  ;                  [lein-exec "0.3.6"]
   ;                 [lein-datomic "0.2.0"]
    ;                [lein-cljsbuild "1.1.3"]
     ;               [lein-fregec "3.24-7.100"]
                    [lein-figwheel "0.5.4-7" :exclusions [org.clojure/core.cache]]]}
                                        ;                    [figwheel-sidecar "0.5.4-6"] :exclusions [org.clojure/core.cache]]
                                        ; figwheel-sidecar break the initial classpath
 :datomic {:install-location "/Users/eklavya/.m2/repository/datomic/datomic-pro-0.9.5394"}}

:repl {
       :plugins [[cider/cider-nrepl "0.14.0"]
                 [refactor-nrepl "2.3.1"]]

       :dependencies [[alembic "0.3.2"]
                      [org.clojure/tools.nrepl "0.2.12"]]}



eklavya-mac/spacemacs.el
;; -*- mode: emacs-lisp -*-
;; This file is loaded by Spacemacs at startup.
;; It must be stored in your home directory.

(defun dotspacemacs/layers ()
  "Configuration Layers declaration.
You should not put any user code in this function besides modifying the variable
values."
  (setq-default
   ;; Base distribution to use. This is a layer contained in the directory
   ;; `+distribution'. For now available distributions are `spacemacs-base'
   ;; or `spacemacs'. (default 'spacemacs)
   dots;pacemacs-distribution 'spacemacs
   ;; Lazy installation of layers (i.e. layers are installed only when a file
   ;; with a supported type is opened). Possible values are `all', `unused'
   ;; and `nil'. `unused' will lazy install only unused layers (i.e. layers
   ;; not listed in variable `dotspacemacs-configuration-layers'), `all' will
   ;; lazy install any layer that support lazy installation even the layers
   ;; listed in `dotspacemacs-configuration-layers'. `nil' disable the lazy
   ;; installation feature and you have to explicitly list a layer in the
   ;; variable `dotspacemacs-configuration-layers' to install it.
   ;; (default 'unused)
   dotspacemacs-enable-lazy-installation 'unused
   ;; If non-nil then Spacemacs will ask for confirmation before installing
   ;; a layer lazily. (default t)
   dotspacemacs-ask-for-lazy-installation t
   ;; If non-nil layers with lazy install support are lazy installed.
   ;; List of additional paths where to look for configuration layers.
   ;; Paths must have a trailing slash (i.e. `~/.mycontribs/')
   dotspacemacs-configuration-layer-path '()
   ;; List of configuration layers to load.
   dotspacemacs-configuration-layers
   '(
     ;; ----------------------------------------------------------------
     ;; Example of useful layers you may want to use right away.
     ;; Uncomment some layer names and press <SPC f e R> (Vim style) or
     ;; <M-m f e R> (Emacs style) to install them.
     ;; ----------------------------------------------------------------
     ivy
     clojure
     auto-completion
     ;; better-defaults
     emacs-lisp
     git
     ;; markdown
     org
     ;; (shell :variables
     ;;        shell-default-height 30
     ;;        shell-default-position 'bottom)
     ;; spell-checking
     ;; syntax-checking
     ;; version-control
     )
   ;; List of additional packages that will be installed without being
   ;; wrapped in a layer. If you need some configuration for these
   ;; packages, then consider creating a layer. You can also put the
   ;; configuration in `dotspacemacs/user-config'.
   dotspacemacs-additional-packages '(

                                      inf-clojure
                                      hy-mode

                                      )
   ;; A list of packages that cannot be updated.
   dotspacemacs-frozen-packages '()
   ;; A list of packages that will not be installed and loaded.
   dotspacemacs-excluded-packages '()
   ;; Defines the behaviour of Spacemacs when installing packages.
   ;; Possible values are `used-only', `used-but-keep-unused' and `all'.
   ;; `used-only' installs only explicitly used packages and uninstall any
   ;; unused packages as well as their unused dependencies.
   ;; `used-but-keep-unused' installs only the used packages but won't uninstall
   ;; them if they become unused. `all' installs *all* packages supported by
   ;; Spacemacs and never uninstall them. (default is `used-only')
   dotspacemacs-install-packages 'used-only))

(defun dotspacemacs/init ()
  "Initialization function.
This function is called at the very startup of Spacemacs initialization
before layers configuration.
You should not put any user code in there besides modifying the variable
values."
  ;; This setq-default sexp is an exhaustive list of all the supported
  ;; spacemacs settings.
  (setq-default
   ;; If non nil ELPA repositories are contacted via HTTPS whenever it's
   ;; possible. Set it to nil if you have no way to use HTTPS in your
   ;; environment, otherwise it is strongly recommended to let it set to t.
   ;; This variable has no effect if Emacs is launched with the parameter
   ;; `--insecure' which forces the value of this variable to nil.
   ;; (default t)
   dotspacemacs-elpa-https t
   ;; Maximum allowed time in seconds to contact an ELPA repository.
   dotspacemacs-elpa-timeout 5
   ;; If non nil then spacemacs will check for updates at startup
   ;; when the current branch is not `develop'. Note that checking for
   ;; new versions works via git commands, thus it calls GitHub services
   ;; whenever you start Emacs. (default nil)
   dotspacemacs-check-for-update nil
   ;; If non-nil, a form that evaluates to a package directory. For example, to
   ;; use different package directories for different Emacs versions, set this
   ;; to `emacs-version'.
   dotspacemacs-elpa-subdirectory nil
   ;; One of `vim', `emacs' or `hybrid'.
   ;; `hybrid' is like `vim' except that `insert state' is replaced by the
   ;; `hybrid state' with `emacs' key bindings. The value can also be a list
   ;; with `:variables' keyword (similar to layers). Check the editing styles
   ;; section of the documentation for details on available variables.
   ;; (default 'vim)
   dotspacemacs-editing-style 'vim
   ;; If non nil output loading progress in `*Messages*' buffer. (default nil)
   dotspacemacs-verbose-loading nil
   ;; Specify the startup banner. Default value is `official', it displays
   ;; the official spacemacs logo. An integer value is the index of text
   ;; banner, `random' chooses a random text banner in `core/banners'
   ;; directory. A string value must be a path to an image format supported
   ;; by your Emacs build.
   ;; If the value is nil then no banner is displayed. (default 'official)
   dotspacemacs-startup-banner 'nil
   ;; List of items to show in startup buffer or an association list of
   ;; the form `(list-type . list-size)`. If nil then it is disabled.
   ;; Possible values for list-type are:
   ;; `recents' `bookmarks' `projects' `agenda' `todos'."
   ;; List sizes may be nil, in which case
   ;; `spacemacs-buffer-startup-lists-length' takes effect.
   dotspacemacs-startup-lists '((recents . 5)
                                (projects . 7))
   ;; True if the home buffer should respond to resize events.
   dotspacemacs-startup-buffer-responsive t
   ;; Default major mode of the scratch buffer (default `text-mode')
   dotspacemacs-scratch-mode 'text-mode
   ;; List of themes, the first of the list is loaded when spacemacs starts.
   ;; Press <SPC> T n to cycle to the next theme in the list (works great
   ;; with 2 themes variants, one dark and one light)
   dotspacemacs-themes '( zenburn
                         spacemacs-light)
   ;; If non nil the cursor color matches the state color in GUI Emacs.
   dotspacemacs-colorize-cursor-according-to-state t
   ;; Default font, or prioritized list of fonts. `powerline-scale' allows to
   ;; quickly tweak the mode-line size to make separators look not too crappy.
   dotspacemacs-default-font '("Hack"
                               :size 15
                               :weight normal
                               :width normal
                               :powerline-scale 1.1)
   ;; The leader key
   dotspacemacs-leader-key "SPC"
   ;; The key used for Emacs commands (M-x) (after pressing on the leader key).
   ;; (default "SPC")
   dotspacemacs-emacs-command-key "SPC"
   ;; The key used for Vim Ex commands (default ":")
   dotspacemacs-ex-command-key ":"
   ;; The leader key accessible in `emacs state' and `insert state'
   ;; (default "M-m")
   dotspacemacs-emacs-leader-key "M-m"
   ;; Major mode leader key is a shortcut key which is the equivalent of
   ;; pressing `<leader> m`. Set it to `nil` to disable it. (default ",")
   dotspacemacs-major-mode-leader-key ","
   ;; Major mode leader key accessible in `emacs state' and `insert state'.
   ;; (default "C-M-m")
   dotspacemacs-major-mode-emacs-leader-key "C-M-m"
   ;; These variables control whether separate commands are bound in the GUI to
   ;; the key pairs C-i, TAB and C-m, RET.
   ;; Setting it to a non-nil value, allows for separate commands under <C-i>
   ;; and TAB or <C-m> and RET.
   ;; In the terminal, these pairs are generally indistinguishable, so this only
   ;; works in the GUI. (default nil)
   dotspacemacs-distinguish-gui-tab nil
   ;; If non nil `Y' is remapped to `y$' in Evil states. (default nil)
   dotspacemacs-remap-Y-to-y$ nil
   ;; If non-nil, the shift mappings `<' and `>' retain visual state if used
   ;; there. (default t)
   dotspacemacs-retain-visual-state-on-shift t
   ;; If non-nil, J and K move lines up and down when in visual mode.
   ;; (default nil)
   dotspacemacs-visual-line-move-text nil
   ;; If non nil, inverse the meaning of `g' in `:substitute' Evil ex-command.
   ;; (default nil)
   dotspacemacs-ex-substitute-global nil
   ;; Name of the default layout (default "Default")
   dotspacemacs-default-layout-name "Default"
   ;; If non nil the default layout name is displayed in the mode-line.
   ;; (default nil)
   dotspacemacs-display-default-layout nil
   ;; If non nil then the last auto saved layouts are resume automatically upon
   ;; start. (default nil)
   dotspacemacs-auto-resume-layouts nil
   ;; Size (in MB) above which spacemacs will prompt to open the large file
   ;; literally to avoid performance issues. Opening a file literally means that
   ;; no major mode or minor modes are active. (default is 1)
   dotspacemacs-large-file-size 1
   ;; Location where to auto-save files. Possible values are `original' to
   ;; auto-save the file in-place, `cache' to auto-save the file to another
   ;; file stored in the cache directory and `nil' to disable auto-saving.
   ;; (default 'cache)
   dotspacemacs-auto-save-file-location 'cache
   ;; Maximum number of rollback slots to keep in the cache. (default 5)
   dotspacemacs-max-rollback-slots 5
   ;; If non nil, `helm' will try to minimize the space it uses. (default nil)
   dotspacemacs-helm-resize nil
   ;; if non nil, the helm header is hidden when there is only one source.
   ;; (default nil)
   dotspacemacs-helm-no-header nil
   ;; define the position to display `helm', options are `bottom', `top',
   ;; `left', or `right'. (default 'bottom)
   dotspacemacs-helm-position 'bottom
   ;; Controls fuzzy matching in helm. If set to `always', force fuzzy matching
   ;; in all non-asynchronous sources. If set to `source', preserve individual
   ;; source settings. Else, disable fuzzy matching in all sources.
   ;; (default 'always)
   dotspacemacs-helm-use-fuzzy 'always
   ;; If non nil the paste micro-state is enabled. When enabled pressing `p`
   ;; several times cycle between the kill ring content. (default nil)
   dotspacemacs-enable-paste-transient-state nil
   ;; Which-key delay in seconds. The which-key buffer is the popup listing
   ;; the commands bound to the current keystroke sequence. (default 0.4)
   dotspacemacs-which-key-delay 0.4
   ;; Which-key frame position. Possible values are `right', `bottom' and
   ;; `right-then-bottom'. right-then-bottom tries to display the frame to the
   ;; right; if there is insufficient space it displays it at the bottom.
   ;; (default 'bottom)
   dotspacemacs-which-key-position 'bottom
   ;; If non nil a progress bar is displayed when spacemacs is loading. This
   ;; may increase the boot time on some systems and emacs builds, set it to
   ;; nil to boost the loading time. (default t)
   dotspacemacs-loading-progress-bar t
   ;; If non nil the frame is fullscreen when Emacs starts up. (default nil)
   ;; (Emacs 24.4+ only)
   dotspacemacs-fullscreen-at-startup t
   ;; If non nil `spacemacs/toggle-fullscreen' will not use native fullscreen.
   ;; Use to disable fullscreen animations in OSX. (default nil)
   dotspacemacs-fullscreen-use-non-native nil
   ;; If non nil the frame is maximized when Emacs starts up.
   ;; Takes effect only if `dotspacemacs-fullscreen-at-startup' is nil.
   ;; (default nil) (Emacs 24.4+ only)
   dotspacemacs-maximized-at-startup nil
   ;; A value from the range (0..100), in increasing opacity, which describes
   ;; the transparency level of a frame when it's active or selected.
   ;; Transparency can be toggled through `toggle-transparency'. (default 90)
   dotspacemacs-active-transparency 90
   ;; A value from the range (0..100), in increasing opacity, which describes
   ;; the transparency level of a frame when it's inactive or deselected.
   ;; Transparency can be toggled through `toggle-transparency'. (default 90)
   dotspacemacs-inactive-transparency 90
   ;; If non nil show the titles of transient states. (default t)
   dotspacemacs-show-transient-state-title t
   ;; If non nil show the color guide hint for transient state keys. (default t)
   dotspacemacs-show-transient-state-color-guide t
   ;; If non nil unicode symbols are displayed in the mode line. (default t)
   dotspacemacs-mode-line-unicode-symbols t
   ;; If non nil smooth scrolling (native-scrolling) is enabled. Smooth
   ;; scrolling overrides the default behavior of Emacs which recenters point
   ;; when it reaches the top or bottom of the screen. (default t)
   dotspacemacs-smooth-scrolling t
   ;; Control line numbers activation.
   ;; If set to `t' or `relative' line numbers are turned on in all `prog-mode' and
   ;; `text-mode' derivatives. If set to `relative', line numbers are relative.
   ;; This variable can also be set to a property list for finer control:
   ;; '(:relative nil
   ;;   :disabled-for-modes dired-mode
   ;;                       doc-view-mode
   ;;                       markdown-mode
   ;;                       org-mode
   ;;                       pdf-view-mode
   ;;                       text-mode
   ;;   :size-limit-kb 1000)
   ;; (default nil)
   dotspacemacs-line-numbers relative
   ;; Code folding method. Possible values are `evil' and `origami'.
   ;; (default 'evil)
   dotspacemacs-folding-method 'evil
   ;; If non-nil smartparens-strict-mode will be enabled in programming modes.
   ;; (default nil)
   dotspacemacs-smartparens-strict-mode nil
   ;; If non-nil pressing the closing parenthesis `)' key in insert mode passes
   ;; over any automatically added closing parenthesis, bracket, quote, etc…
   ;; This can be temporary disabled by pressing `C-q' before `)'. (default nil)
   dotspacemacs-smart-closing-parenthesis nil
   ;; Select a scope to highlight delimiters. Possible values are `any',
   ;; `current', `all' or `nil'. Default is `all' (highlight any scope and
   ;; emphasis the current one). (default 'all)
   dotspacemacs-highlight-delimiters 'all
   ;; If non nil, advise quit functions to keep server open when quitting.
   ;; (default nil)
   dotspacemacs-persistent-server nil
   ;; List of search tool executable names. Spacemacs uses the first installed
   ;; tool of the list. Supported tools are `ag', `pt', `ack' and `grep'.
   ;; (default '("ag" "pt" "ack" "grep"))
   dotspacemacs-search-tools '("ag" "pt" "ack" "grep")
   ;; The default package repository used if no explicit repository has been
   ;; specified with an installed package.
   ;; Not used for now. (default nil)
   dotspacemacs-default-package-repository nil
   ;; Delete whitespace while saving buffer. Possible values are `all'
   ;; to aggressively delete empty line and long sequences of whitespace,
   ;; `trailing' to delete only the whitespace at end of lines, `changed'to
   ;; delete only whitespace for changed lines or `nil' to disable cleanup.
   ;; (default nil)
   dotspacemacs-whitespace-cleanup nil
   ))

(defun dotspacemacs/user-init ()
  "Initialization function for user code.
It is called immediately after `dotspacemacs/init', before layer configuration
executes.
 This function is mostly useful for variables that need to be set
before packages are loaded. If you are unsure, you should try in setting them in
`dotspacemacs/user-config' first."
  )

(defun dotspacemacs/user-config (





                                 )
  "Configuration function for user code.
This function is called at the very end of Spacemacs initialization after
layers configuration.
This is the place where most of your configurations should be done. Unless it is
explicitly specified that a variable should be set before a package is loaded,
you should place your code here."
  )

;; Do not write anything past this comment. This is where Emacs will
;; auto-generate custom variable definitions.
(custom-set-variables
 ;; custom-set-variables was added by Custom.
 ;; If you edit it by hand, you could mess it up, so be careful.
 ;; Your init file should contain only one such instance.
 ;; If there is more than one, they won't work right.
 '(cider-default-repl-command "boot")
 '(package-selected-packages
   (quote
    (inf-clojure hy-mode clojure-snippets clj-refactor inflections edn multiple-cursors paredit peg cider-eval-sexp-fu cider queue clojure-mode smeargle orgit org-projectile org-category-capture org-present org-pomodoro alert log4e gntp org-download magit-gitflow htmlize gnuplot gitignore-mode gitconfig-mode gitattributes-mode git-timemachine git-messenger git-link fuzzy evil-magit magit magit-popup git-commit with-editor company-statistics company auto-yasnippet yasnippet ac-ispell auto-complete wgrep smex ivy-hydra counsel-projectile counsel swiper ivy zenburn-theme ws-butler winum which-key volatile-highlights vi-tilde-fringe uuidgen use-package toc-org spaceline restart-emacs request rainbow-delimiters popwin persp-mode pcre2el paradox org-plus-contrib org-bullets open-junk-file neotree move-text monokai-theme macrostep lorem-ipsum linum-relative link-hint info+ indent-guide hungry-delete hl-todo highlight-parentheses highlight-numbers highlight-indentation hide-comnt help-fns+ helm-themes helm-swoop helm-projectile helm-mode-manager helm-make helm-flx helm-descbinds helm-ag google-translate golden-ratio flx-ido fill-column-indicator fancy-battery eyebrowse expand-region exec-path-from-shell evil-visualstar evil-visual-mark-mode evil-unimpaired evil-tutor evil-surround evil-search-highlight-persist evil-numbers evil-nerd-commenter evil-mc evil-matchit evil-lisp-state evil-indent-plus evil-iedit-state evil-exchange evil-escape evil-ediff evil-args evil-anzu eval-sexp-fu elisp-slime-nav dumb-jump define-word column-enforce-mode clean-aindent-mode auto-highlight-symbol auto-compile aggressive-indent adaptive-wrap ace-window ace-link ace-jump-helm-line))))
(custom-set-faces
 ;; custom-set-faces was added by Custom.
 ;; If you edit it by hand, you could mess it up, so be careful.
 ;; Your init file should contain only one such instance.
 ;; If there is more than one, they won't work right.
 '(default ((t (:foreground "#DCDCCC" :background "#3F3F3F")))))
